examples/slash_commands/main.go

```diff
@@ -1,386 +1,60 @@
 package main
 
 import (
-	"flag"
-	"fmt"
 	"log"
 	"os"
 	"os/signal"
-	"strings"
-	"time"
+	"syscall"
 
 	"github.com/bwmarrin/discordgo"
 )
 
-// Bot parameters
-var (
-	GuildID        = flag.String("guild", "", "Test guild ID. If not passed - bot registers commands globally")
-	BotToken       = flag.String("token", "", "Bot access token")
-	RemoveCommands = flag.Bool("rmcmd", true, "Remove all commands after shutdowning or not")
-)
+// If you are getting "type not declared by package" error,
+// try executing "go get github.com/bwmarrin/discordgo@master".
+// It happens because "go get" pulls package's latest release instead
+// of the latest commit.
 
-var s *discordgo.Session
+const GuildID string = "Bot token goes here"
+const BotToken string = "Guild where the bot will be used"
```

Comment:
I'm not sure if this is a good idea, since most of the time sensitive stuff is put into flags, plus it is easy to replace these values without recompiling the application again.